app/src/main/res/values/color.xml

```diff
@@ -5,10 +5,7 @@
     <color name="colorAccent">@color/accent</color>
 
     <!-- Colors from Palette -->
-    <color name="primary">#207be6</color>
-    <color name="primary_dark">#195ed2</color>
     <color name="primary_light">#BBDEFB</color>
-    <color name="accent">#FF5300</color>
```

Comment:
Maybe keep color `accent,primary,primary_dark` (for the default values) and reference the color behind it
`<color name="accent">@color/md_deep_orange_500</color>`